### PR TITLE
feat(reuse): REUSE best practices for licensing/copyright part-4

### PR DESCRIPTION
**Pull request by rohitpandey49**
Signed-off-by: Rohit Pandey <rohit.pandey4900@gmail.com>

## Description

REUSE best practices for licensing/copyright

### Changes

1. Implemented correct licensing/copyright format

### Current REUSE status

When running reuse lint over this branch, the result is as follows:

```
* Files with copyright information: 4364 / 4364
* Files with license information: 4364 / 4364
```

Before that we were at:

```
* Files with copyright information: 4152 / 4360
* Files with license information: 3723 / 4360
```

<a href="https://gitpod.io/#https://github.com/fossology/fossology/pull/2263"></a>

### CODE REVIEW

**Comment by GMishx** [comment]
Please change the license of `NOTICES.spdx` from `GPL-2.0-only` to `CC0-1.0` as written in the file.

File: src/www/ui/core-schema.dat [hunk 1]
```diff
@@ -1,4 +1,7 @@
 <?php
+# SPDX-FileCopyrightText: © Fossology contributors
+
+# SPDX-License-Identifier: GPL-2.0-only
```
Comment (GMishx) [inline src/www/ui/core-schema.dat:4]: It is a PHP file
```suggestion
/*
 SPDX-FileCopyrightText: © Fossology contributors

 SPDX-License-Identifier: GPL-2.0-only
*/
```

File: src/www/ui/scripts/jquery.treeview.js [hunk 2]
```diff
@@ -4,11 +4,9 @@
...
  * http://docs.jquery.com/Plugins/Treeview
  *
- * Copyright (c) 2007 Jörn Zaefferer
- *
- * Dual licensed under the MIT and GPL licenses:
- *   http://www.opensource.org/licenses/mit-license.php
- *   http://www.gnu.org/licenses/gpl.html
+ * SPDX-FileCopyrightText: © 2007 Jörn Zaefferer
+ * 
+ * SPDX-License-Identifier: MIT AND GPL-3.0-only
```
Comment (GMishx) [inline src/www/ui/scripts/jquery.treeview.js:9]:
```suggestion
 * SPDX-License-Identifier: MIT AND GPL-2.0-only
```

File: src/www/ui/scripts/jquery.validate.js [hunk 3]
```diff
@@ -4,13 +4,12 @@
  * http://bassistance.de/jquery-plugins/jquery-plugin-validation/
  * http://docs.jquery.com/Plugins/Validation
  *
- * Copyright (c) 2006 - 2008 Jörn Zaefferer
+ * SPDX-FileCopyrightText: © 2006 - 2008 Jörn Zaefferer
+ * 
+ * SPDX-License-Identifier: MIT AND GPL-3.0-only
```
Comment (GMishx) [inline src/www/ui/scripts/jquery.validate.js:9]:
```suggestion
 * SPDX-License-Identifier: MIT AND GPL-2.0-only
```

File: src/www/ui/template/admin-folder-edit-form.html.twig [hunk 4]
```diff
@@ -1,3 +1,7 @@
+{# SPDX-FileCopyrightText: © ossology contributors
```
Comment (GMishx) [inline src/www/ui/template/admin-folder-edit-form.html.twig:1]:
```suggestion
{# SPDX-FileCopyrightText: © FOSSology contributors
```

**Review by GMishx — CHANGES_REQUESTED** [review]

**Comment by rohitpandey49** [comment]
Corrected @GMishx sir

**Review by GMishx — CHANGES_REQUESTED** [review]
For `NOTICES.spdx`, there are several unnecessary changes. Can you please revert the file content and maybe use some basic editors like [GEdit](https://wiki.gnome.org/Apps/Gedit) in Ubuntu or [VIM](https://www.vim.org/) to modify the file?

**Comment by rohitpandey49** [comment]
> For `NOTICES.spdx`, there are several unnecessary changes. Can you please revert the file content and maybe use some basic editors like [GEdit](https://wiki.gnome.org/Apps/Gedit) in Ubuntu or [VIM](https://www.vim.org/) to modify the file?

Updated sir.

**Review by GMishx — APPROVED** [review]
Changes looks good.

**Pull request merged**